### PR TITLE
feat: adding --with-raw-capability flag

### PR DESCRIPTION
**Pull request by spex66**
- for 'deploy' and 'diagram' command (so the parameter comes after the command)
- addressing issue #28 

examples
```bash
# with  global parameter dry-run
➟  poetry run bootstrap-cli --dry-run yes deploy --with-raw-capability no .local/config-simple.yml 
# deploy w/o raw
➟  poetry run bootstrap-cli deploy --with-raw-capability no .local/config-simple.yml 
# diagram w/o raw
➟  poetry run bootstrap-cli diagram --with-raw-capability no .local/config-simple.yml 
```

### CODE REVIEW

File: incubator/bootstrap_cli/__main__.py [hunk 1]
```diff
@@ -1059,9 +1071,20 @@ def deploy(self, with_special_groups: YesNoType = YesNoType.no):
         _logger.debug(f"GROUPS in CDF:\n{self.deployed['groups']}")
 
         # run generate steps (only print results atm)
-        target_raw_dbs, new_created_raw_dbs = self.generate_missing_raw_dbs()
-        _logger.info(f"All RAW_DBS from config:\n{target_raw_dbs}")
-        _logger.info(f"New RAW_DBS to CDF:\n{new_created_raw_dbs}")
+
+        target_raw_dbs: List[str] = []
+        new_created_raw_dbs: List[str] = []
+        if self.with_raw_capability:
```
Comment (sdorheim) [inline incubator/bootstrap_cli/__main__.py:1077]: could it be helpful to this on a core- or namespace-level? I could envision a lot of "uc" level scopes not needing raw-dbs?

**Review by gaetan-h — APPROVED** [review]
Looks good and tested successfully

File: incubator/bootstrap_cli/__main__.py (hunk 1 above)
Comment (spex66) [reply incubator/bootstrap_cli/__main__.py:1077]: as it is a (hopefully) temporary bypass for a the RAW DB limit of 32characters, I would like to keep it on a global-level

Comment (spex66) [reply incubator/bootstrap_cli/__main__.py:1077]: more clear: 
* I definitive want to keep a "templated / homogeneous / packaged" approach over any option to make detailed customization
=> in my opinion: as easier the setup is to understand, as more trust you can create in the access-model

**Pull request merged**